bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -553,13 +677,12 @@ public void actionPerformed(final ActionEvent e) {
     private final ActionListener autoGuideStarListener = new ActionListener() {
         @Override public void actionPerformed (final ActionEvent evt) {
             try {
+                // TODO: For BAGS, we do not want to pop open the TPE.
```

Comment:
Yeah, it will, but this is the big step between what we had before and full on BAGS... so for now, we still require users to click the auto button.

Our next step is to shift everything over to the BagsManager and remove the auto buttons.